project.yaml

```diff
@@ -0,0 +1,4805 @@
+                      "asset_type": "AssetType",
+                      "primary_issuance_agent": "Option<IdentityId>",
+                    },
+                  "PalletName": "Text",
+                  "DispatchableName": "Text",
+                  "PalletPermissions":
+                    {
+                      "pallet_name": "PalletName",
+                      "dispatchable_names": "Option<Vec<DispatchableName>>",
+                    },
+                  "Permissions":
+                    {
+                      "asset": "Option<Vec<Ticker>>",
+                      "extrinsic": "Option<Vec<PalletPermissions>>",
+                      "portfolio": "Option<Vec<PortfolioId>>",
+                    },
+                  "LegacyPalletPermissions":
+                    {
+                      "pallet_name": "PalletName",
+                      "total": "bool",
+                      "dispatchable_names": "Vec<DispatchableName>",
+                    },
+                  "LegacyPermissions":
+                    {
+                      "asset": "Option<Vec<Ticker>>",
+                      "extrinsic": "Option<Vec<LegacyPalletPermissions>>",
+                      "portfolio": "Option<Vec<PortfolioId>>",
+                    },
+                  "Signatory":
+                    {
+                      "_enum":
+                        { "Identity": "IdentityId", "Account": "AccountId" },
+                    },
+                  "SecondaryKey":
+                    { "signer": "Signatory", "permissions": "Permissions" },
+                  "SecondaryKeyWithAuth":
+                    {
+                      "secondary_key": "SecondaryKey",
+                      "auth_signature": "Signature",
+                    },
+                  "IdentityRole":
+                    {
+                      "_enum":
+                        [
+                          "Issuer",
+                          "SimpleTokenIssuer",
+                          "Validator",
+                          "ClaimIssuer",
+                          "Investor",
+                          "NodeRunner",
+                          "PM",
+                          "CDDAMLClaimIssuer",
+                          "AccreditedInvestorClaimIssuer",
+                          "VerifiedIdentityClaimIssuer",
+                        ],
+                    },
+                  "PreAuthorizedKeyInfo":
+                    {
+                      "target_id": "IdentityId",
+                      "secondary_key": "SecondaryKey",
+                    },
+                  "DidRecord":
+                    {
+                      "primary_key": "AccountId",
+                      "secondary_keys": "Vec<SecondaryKey>",
+                    },
+                  "KeyIdentityData":
+                    {
+                      "identity": "IdentityId",
+                      "permissions": "Option<Permissions>",
+                    },
+                  "CountryCode":
```

Comment:
Why it is defined multiple times ?

Reply:
I take the `polymesh_schema`.json from each version and add it directly, diffing it would be error-prone

Reply:
Okay